starter-project-web-vue2/pages/sileshi/index.vue

```diff
@@ -0,0 +1,20 @@
+<template>
+  <v-app>
+    <v-main class="white">
+      <BlogPost />
+    </v-main>
+  </v-app>
+</template>
+
+<script>
+import BlogPost from '../../components/sileshi/SileshiBlogPost.vue'
+export default {
+  name: 'IndexPage',
+  components: { BlogPost },
+}
+</script>
+<style>
+.padd {
+  padding-top: 5rem;
```

Comment:
also here we can use vuetify classes. eg. pt-2, refer [here](https://vuetifyjs.com/en/styles/spacing/) for more